ykman/util.py

```diff
@@ -76,6 +76,26 @@ def dependent_on_ccid():
         return CAPABILITY.OPGP | CAPABILITY.OATH | CAPABILITY.PIV
 
 
+@unique
+class FORM_FACTOR(IntEnum):
+    UNKNOWN = 0x00
+    USB_A_KEYCHAIN = 0x01
+    USB_A_NANO = 0x02
+    USB_C_KEYCHAIN = 0x03
+    USB_C_NANO = 0x04
+
+    @classmethod
+    def printable(cls, form_factor):
```

Comment:
I would override `__str__(self) ` instead, so that you can just print a value and have it look good, without having to explicitly call printable.